scheduler/src/cook/mesos/api.clj

```diff
@@ -844,11 +844,23 @@
             application (assoc :application (util/remove-datomic-namespacing application))
             expected-runtime (assoc :expected-runtime expected-runtime))))
 
-(defn fetch-group-job-details
+(defn fetch-group-jobs-by-state
```

Comment:
Missing docstring

Reply:
I removed this function when I refactored to use `filter` based on your other suggestion.